src/Adapters/__mockData__/DemoEnvsConfig.json

```diff
@@ -1,1663 +1,1592 @@
-            "expressionType": "CategoricalValues",
-            "valueExpression": "PrimaryTwin.CO2Level > 40",
-            "valueRanges": [
-              {
-                "id": "01525277c517c8062b164bd6447be49a",
-                "values": [
-                  true
+                "behaviorIDs": ["83f16294b7959c4e362aa4c1c3264f85"],
+                "latitude": 37.8044,
+                "longitude": -122.2712
+            },
+            {
+                "id": "88112b3417e053c276e9f441ec9270a9",
+                "displayName": "Robotic Arms",
+                "description": "TMG Headquarters",
+                "assets": [
+                    {
+                        "type": "3DAsset",
+                        "url": "https://iotcardboardjsresources.blob.core.windows.net/public/RobotArms.glb"
+                    }
                 ],
-                "visual": {
-                  "color": "#EE92C2",
-                  "iconName": "Shield",
-                  "labelExpression": "`CO2 Level Dangerously High`"
-                }
-              }
-            ],
-            "objectIDs": {
-              "expression": "objectIDs"
-            }
-          },
-          {
-            "type": "Popover",
-            "title": "",
-            "widgets": [
-              {
-                "id": "5faa9077a55ecc616b0e3d14710646a8",
-                "type": "Value",
-                "widgetConfiguration": {
-                  "displayName": "CO2 Level",
-                  "valueExpression": "PrimaryTwin.CO2Level",
-                  "type": "double"
-                }
-              },
-              {
-                "id": "9de1f028aee9f7eefaa0ae05d541b81c",
-                "type": "Link",
-                "widgetConfiguration": {
-                  "label": "Mitigation Tactics",
-                  "linkExpression": "`https://contoso.com/co2mitigation`"
-                }
-              }
-            ],
-            "objectIDs": {
-              "expression": "meshIDs"
-            }
-          }
-        ]
-      },
-      {
-        "id": "d0fdbfdde9a5c6f1bfb2c16738722885",
-        "displayName": "Humidity Level",
-        "datasources": [
-          {
-            "type": "ElementTwinToObjectMappingDataSource",
-            "elementIDs": [
-              "29ac7cca3a22a06784fd2c9d1e45ef1a",
-              "9fead963403f6d16f1b95068e18fb5da",
-              "3a9ad613f55a4dc952b5da9e3e3ed545",
-              "9faeafa393d1e6463877c939f862487c",
-              "f87e265e8772992c941c2155195c33fc",
-              "94e06fe54f1102bd8dc855cc428c1981",
-              "2f8016a04a9a44c2fd4546f85c50761f"
-            ]
-          }
-        ],
-        "visuals": [
-          {
-            "type": "ExpressionRangeVisual",
-            "expressionType": "NumericRange",
-            "valueExpression": "PrimaryTwin.Humidity",
-            "valueRanges": [
-              {
-                "values": [
-                  0,
```

Comment:
What are we doing to the demo environments?

Reply:
+1, not clear why this needed to be changed?

Reply:
wow, this diff looks weird I only added data history widgets for demo scenes. Anyways, I will take the changes back for this file.

Reply:
I like the idea of including a chart on the demos, let's just see if we can minimize the changes there.

Reply:
I think it was because of linting, okay I can try again

Reply:
of course messing the demo env was not my intention haha

Reply:
yeah it is linting so you can ignore the diff counts, only differences is data history widget visuals:
![image](https://user-images.githubusercontent.com/45217314/198728046-78e4eae0-49e6-46e1-a0fe-f695fea0d6f5.png)